vm_manager_cmd.py

```diff
@@ -228,6 +228,36 @@ def __call__(self, parser, namespace, values, option_string=None):
                 default=None,
                 help="Sets a priority for this guest",
             )
+            p.add_argument(
+                "--pacemaker-meta",
+                type=str,
+                metavar="key=value",
+                required=False,
+                help='Set a number of key-value pacemaker "meta"'
```

Comment:
"Set a number of key-value" is confusing and not clear.

Something like "Set a key-value pacemaker "meta". Can be used multiple times [...]" would clearly state that you set the key/value pair and that you can use it more than once.

Also do it for the `--pacemaker-params` and `--pacemaker-utilization` options, as well as for your commit message.